src/test/java/CPPParserTest.java

```diff
@@ -1,152 +1,155 @@
+                new TestCase("#if __has_c_attribute (nonnull) && A", "HAS_C_ATTRIBUTE___LB__nonnull__RB__&&A"),
+
+                new TestCase("#if defined __has_builtin", "DEFINED_HAS_BUILTIN_"),
+                new TestCase("#if __has_builtin", "HAS_BUILTIN_"),
+                new TestCase("#if __has_builtin (__nonnull)", "HAS_BUILTIN___LB____nonnull__RB__"),
+                new TestCase("#if __has_builtin (nonnull) && A", "HAS_BUILTIN___LB__nonnull__RB__&&A"),
+
+                new TestCase("#if A // Comment && B", "A"),
+                new TestCase("#if A /* Comment */ && B", "A&&B"),
+                new TestCase("#if A && B /* Multiline Comment", "A&&B"),
+
+                new TestCase("#if A == B", "A__EQ__B"),
+                new TestCase("#if A == 1", "A__EQ__1"),
+
+                new TestCase("#if defined A", "DEFINED_A"),
+                new TestCase("#if defined(A)", "DEFINED___LB__A__RB__"),
+                new TestCase("#if defined (A)", "DEFINED___LB__A__RB__"),
+                new TestCase("#if defined ( A )", "DEFINED___LB__A__RB__"),
+                new TestCase("#if (defined A)", "(DEFINED_A)"),
+                new TestCase("#if MACRO (A)", "MACRO___LB__A__RB__"),
+                new TestCase("#if MACRO (A, B)", "MACRO___LB__A__B__RB__"),
+                new TestCase("#if MACRO (A, B + C)", "MACRO___LB__A__B__ADD__C__RB__"),
+                new TestCase("#if MACRO (A, B) == 1", "MACRO___LB__A__B__RB____EQ__1"),
+
+                new TestCase("#if ifndef", "ifndef"),
+
+                new TestCase("#if __has_include_next(<some-header.h>)", "__HAS_INCLUDE_NEXT___LB____LT__some__SUB__header__DOT__h__GT____RB__"),
+                new TestCase("#if __is_target_arch(x86)", "__IS_TARGET_ARCH___LB__x86__RB__"),
+                new TestCase("#if A || (defined(NAME) && (NAME >= 199630))", "A||(DEFINED___LB__NAME__RB__&&(NAME__GEQ__199630))"),
+                new TestCase("#if MACRO(part:part)", "MACRO___LB__part__COLON__part__RB__"),
+                new TestCase("#if MACRO(x=1)", "MACRO___LB__x__ASSIGN__1__RB__"),
+                new TestCase("#if A = 3", "A__ASSIGN__3"),
+                new TestCase("#if ' ' == 32", "__SQUOTE_____SQUOTE____EQ__32"),
+                new TestCase("#if (NAME<<1) > (1<<BITS)", "__LB__NAME__LSHIFT__1__RB____GT____LB__1__LSHIFT__BITS__RB__"),
+                new TestCase("#if #cpu(sparc)", "CPU___LB__sparc__RB__"),
+                new TestCase("#ifdef \\U0001000", "__B_SLASH__U0001000"),
+                new TestCase("#if (defined(NAME) && (NAME >= 199905) && (NAME < 1991011)) ||     (NAME >= 300000) || defined(NAME)", "(DEFINED___LB__NAME__RB__&&(NAME__GEQ__199905)&&(NAME__LT__1991011))||(NAME__GEQ__300000)||DEFINED___LB__NAME__RB__"),
+                new TestCase("#if __has_warning(\"-Wa-warning\"_foo)",
+                        "__HAS_WARNING___LB____QUOTE____SUB__Wa__SUB__warning__QUOTE_____foo__RB__")
         );
```

Comment:
Why is this reindentation necessary? Why should we use 8 spaces instead of 4 spaces now? (Next time I change something in IntelliJ, this styling will be reset and we have huge patches again. 😵 )

Reply:
I have not used nvim because I was worried about my style being applied to reformat the entire codebase 🙈 . I only used IntelliJ which did this autoformatting.

Reply:
Oh I see. Seems we have different settings then. I guess its time for a common .editorconfig then. I will open an issue soon then. You dont have to revert the indentation stuff. That is too cumbersome I guess.

Reply:
Have you tried formatting he file with your Intellij? I am using the default settings, because I only just installed it on my new Laptop. Perhaps something else in the implementation had changed since the last time it was formatted?

Reply:
I just formatted this block of TestCases in IntelliJ and the same formatting as you have was applied. Interesting...